addons/payment_razorpay/tests/test_payment_transaction.py

```diff
@@ -50,6 +50,27 @@ def test_void_is_not_supported(self):
         tx = self._create_transaction('direct', state='authorized')
         self.assertRaises(UserError, func=tx._send_void_request)
 
+    def test_prevent_multi_payments_on_recurring_transactions(self):
+        """ Test that no retry is allowed within the 36 hours of the first attempt using token. """
+        shared_token = self._create_token()
+        tx1 = self._create_transaction('token', reference='INV123', token_id=shared_token.id)
+        tx1.state = 'pending'
```

Comment:
You can pass `state='pending'` directly to `_create_transaction`.

Reply:
I did not saw at first I could pass any value to the create transaction of tests. Thanks :+1: